openapi/components/schemas/GatewayAccountConfig/Pagsmile.yaml

```
@@ -16,10 +16,25 @@ allOf:
             description: ID of the Pagsmile application.
           secretKey:
             type: string
-            description: Pagsmile secret key.
+            description: Pagsmile MD5 key.
+            format: password
+            writeOnly: true
+          apiSecretKey:
+            type: string
+            description: Pagsmile API secret key start with Pagsmile_sk.
```

suggestion: suggestion for readability
```suggestion
            description: Pagsmile API secret key. This key starts with `Pagsmile_sk`.
```